config/crd/bases/rbac-operator.argoproj-labs.io_argocdrolebindings.yaml

```diff
@@ -40,6 +40,7 @@ spec:
             description: ArgoCDRoleBindingSpec defines the desired state of ArgoCDRoleBinding
             properties:
               argocdRoleRef:
+                description: argocdRoleRef defines the reference to the role being granted.
```

Comment:
please also add to api/v1alpha1/argocdrolebinding_types.go

Reply:
done!